src/constants.ts

```diff
@@ -43,6 +43,7 @@ export enum APP_PATHS {
   MONITORING = '/monitoring',
   MODULES_FOR_PROJECT = '/modules/project/:projectId',
   MODULES_FOR_PROJECT_CONTENT = '/modules/project/:projectId/content/:moduleId',
+  MODULES_FOR_PROJECT_EVENT = '/modules/project/:projectId/content/:moduleId/event/:eventId',
```

Comment:
Made a guess here, will adjust if we'd prefer a different path.

Reply:
Likely we can just use `/modules/project/:projectId/event:eventId`

Because we probably won't use `moduleId` to query for the full event details. But I do like the this path showing the hierarchy of things.